### ISSUE TITLE
Interfaces / network adapters display broken on Linux

### ISSUE THREAD

**Issue by Allowee** [Ylianst/MeshCentral#1557]
I know this has been changed recently, but I can't get the Interfaces link on 'General' tab to work. Hangs on "Loading..."
Also on the details tab no mention of network cards.

JS console error message:
Uncaught TypeError: Cannot read property 'startsWith' of undefined
    at updateDeviceDetails (VM1962:8168)
    at Object.onMessage (VM1962:2136)
    at WebSocket.MeshServerCreateControl.obj.xxOnMessage (meshcentral.js:52)

part of 'netinfo' on the non working agent, Debian 9 system.

```
 enp6s0: {
    0: {
      mac: "00:02:A5:45:75:6A"
      type: "ethernet"
      status: "up"
      address: "192.168.38.1"
      family: "IPv4"
      netmask: "255.255.255.0"
    }
  }
  ```
 
same mesh server, Windows 10 agent which works fine:

```
   LAN-verbinding: {
    0: {
      address: "fe80::306a:bad1:256a:d118%13"
      fqdn: ""
      family: "IPv6"
      mac: "D4:85:64:05:18:E9"
      index: 13
      status: "up"
      type: "ethernet"
    }
    1: {
      address: "192.168.28.61"
      gateway: "192.168.28.1"
      fqdn: ""
      family: "IPv4"
      netmask: "255.255.255.0"
      mac: "D4:85:64:05:18:E9"
      index: 13
      status: "up"
      type: "ethernet"
    }
  }
```

**Comment by Ylianst** [comment]
I just did some fixing for this, but not sure what is causing the problem but hopefully MeshCentral v0.5.76 should fix it.

**Comment by Ylianst** [comment]
Just published MeshCentral v0.5.76 with (hopefully) the fix for this.

**Comment by Allowee** [comment]
Yes the interfaces dialog is working again and interfaces are shown in the details tab.

Thank you for the quick fix!
